StudentManagement/StudentManagement.Console/Program.cs

```diff
@@ -1,15 +1,61 @@
 ﻿using StudentManagement.Console;
+bool menu = false;
+var students = new List<Student>();
+var student = new Student();
+do
+{
+
+
+    Console.WriteLine("1 -- Insert new Student\t");
+    Console.WriteLine("2 -- Show all students\t");
+    Console.WriteLine("Choose");
+    int option = int.Parse(Console.ReadLine());
+   
+
+
+    
+    switch (option)
+    {
+        case 1:
+            Console.WriteLine("Enter student name");
+            student.Name = Console.ReadLine();
+            Console.WriteLine("Enter student subname");
+            student.Surname = Console.ReadLine();
+            Console.WriteLine("What the student is studying");
+            student.Occupation = Console.ReadLine();
+            Console.WriteLine("Enter student id");
+            student.Id = int.Parse(Console.ReadLine());
+            Console.WriteLine("How old is student");
+            student.Age = int.Parse(Console.ReadLine());
+            students.Add(student);
+            break;
+        case 2:
+            foreach (Student studentList in students)
```

Comment:
1. you can use var here
2. you're iterating over students - how you can call a single student "studentList". It's a "student"

Reply:
foreach loop is way better when i have to call the objects from list. Thank you Konrad